lib/Jsrt/Jsrt.cpp

```diff
@@ -2511,6 +2512,92 @@ CHAKRA_API JsHasException(_Out_ bool *hasException)
     return JsNoError;
 }
 
+CHAKRA_API JsGetAndClearExceptionWithMetadata(_Out_ JsValueRef *metadata)
```

Comment:
Apart from the return part, this implementation looks like an exact match to `JsGetAndClearException` implementation below. In order not to duplicate, could it be better to define a common function and make these two public methods using that?